.changeset/sharp-chairs-report.md

```diff
@@ -0,0 +1,14 @@
+---
+'skuba': major
+---
+
+deps: Require Node.js 18.12+
+
+Node.js 16 will reach end of life by September 2023. We have aligned our version support with [sku 18](https://github.com/seek-oss/sku/releases/tag/sku%4012.0.0).
```

Comment:
I noticed this in Renovate PRs.

```suggestion
Node.js 16 will reach end of life by September 2023. We have aligned our version support with [sku 12](https://github.com/seek-oss/sku/releases/tag/sku%4012.0.0).
```

The horse has already bolted, but the changelog and release notes can still be manually edited to fix the typo.